CONTRIBUTING.md

```diff
@@ -251,15 +251,16 @@ and follow [those instructions](https://github.com/elastic/docs#for-a-local-repo
 If you have access to make releases, the process is as follows:
 
 1. Check if sonatype is up: https://status.maven.org
-1. Review project version. The release version will be `${project.version}` without the `-SNAPSHOT`. 
+1. Review project version. The release version will be `${project.version}` without the `-SNAPSHOT`.
    1. In case you want to update the version, execute `mvn release:update-versions`
 1. Execute the release Jenkins job on the internal ci server. This job is same as the snapshot-build job, but it also:
    1. Removes `-SNAPSHOT` from all `${project.version}` occurrences and makes a commit before build
    1. Tags this new commit with the version name, e.g. `v1.1.0`
    1. Advances the version for next development iteration and makes a commit
    1. Uploads artifacts to maven central
+   1. (Optional) If the `push_docker` parameter is set, build and push a Docker image for this release to the docker.elastic.co repo.
```

Comment:
IIUC, this step is not for the https://github.com/elastic/apm-agent-java/pull/1001/files#diff-6a3371457528722a734f3c51d9238c13R256 but for another Jenkins instance

Reply:
That's true for the time being. We are hoping that soon we can migrate everything over here which is why I left this in but after looking at this, I think it could mislead people. I'll remove it. Thanks!